### PR TITLE
block creating server cert until the certificate authority is available

### PR DESCRIPTION
**Pull request by addyess**
[LP 1992676](https://bugs.launchpad.net/charm-easyrsa/+bug/1992676)

### CODE REVIEW

**Comment by Cynerva** [comment]
I suspect create_client_cert is going to need the same guard

**Review by Cynerva — APPROVED** [review]
LGTM but after this lands, I think you might hit a similar failure on create_client_cert. I would consider adding the same guard there.

**Comment by addyess** [comment]
lemme just go ahead an add it...

**Pull request merged**
